lib/player_factory.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'human_player'
+require_relative 'computer_player'
+
+module TicTacToe
+  module PlayerFactory
+    def self.build(config)
+      shared_parameters = {player_mark: config[:player_mark]}
+
+      case config[:type]
+      when :human
```

Comment:
I would use constants for these sort of values (`:human`, `:computer`). Right now, they are likely duplicated in few places and are what are referred to as "Magic Strings."